libmobilecoin/src/transaction.rs

```diff
@@ -147,15 +173,15 @@ pub extern "C" fn mc_tx_out_get_value(
     out_error: FfiOptMutPtr<FfiOptOwnedPtr<McError>>,
 ) -> bool {
     ffi_boundary_with_error(out_error, || {
-        let amount = Amount::try_from_ffi(&tx_out_amount)?;
         let tx_out_public_key = RistrettoPublic::try_from_ffi(&tx_out_public_key)?;
         let view_private_key = RistrettoPrivate::try_from_ffi(&view_private_key)
             .expect("view_private_key is not a valid RistrettoPrivate");
 
         let shared_secret = get_tx_out_shared_secret(&view_private_key, &tx_out_public_key);
-        let (val, _blinding) = amount.get_value(&shared_secret)?;
```

Comment:
so, because we are removing this line, we no longer have a failure mode that corresponds to "TransactionCrypto", I think this is why kyle wanted to have the `crc32` value that we check

OTOH I think we only get to this point after view key matching has succeeded?

You might want to update documentation that TransactionCrypto is not a failure mode anymore?

Reply:
How would I check the crc32 of this amount ? Would I need to pass in the original TxOut object from the client ?